### PR TITLE
KAFKA-15311: Fix docs about reverting to ZooKeeper mode during KRaft migration

### PR DESCRIPTION
**Pull request by lzyLuke**
Pick up works left from https://github.com/apache/kafka/pull/14160/files

### CODE REVIEW

File: docs/ops.html [hunk 1]
```diff
@@ -3641,7 +3641,6 @@ <h3>Limitations</h3>
       using one of those features, you will not be able to migrate to KRaft yet.</li>
```
Comment (ex172000) [inline docs/ops.html]: Maybe we can phrase like this here: `If you are using one of those features, you are not able to migrate to KRaft yet.`

File: docs/ops.html [hunk 2]
```diff
@@ -3787,7 +3786,7 @@ <h3>Migrating brokers to KRaft</h3>
   <h3>Reverting to ZooKeeper mode During the Migration</h3>
     While the cluster is still in migration mode, it is possible to revert to ZK mode. In order to do this:
     <ol>
-      <li>One by one, take each KRaft broker down. Remove the __cluster_metadata directory on the broker. Then, restart the broker as ZooKeeper.</li>
+      <li>One by one, take each KRaft broker down. Remove the __cluster_metadata directory on the broker. Then, restart the broker as zookeeper mode.</li>
```
Comment (ex172000) [inline docs/ops.html]: `with` zookeeper mode?

**Review by ex172000 — APPROVED** [review]

File: docs/ops.html (hunk 2 above)
Comment (lzyLuke) [reply docs/ops.html]: maybe `in` would be better :)

File: docs/ops.html [hunk 3]
```diff
@@ -3613,23 +3613,22 @@ <h4 class="anchor-heading"><a id="kraft_zk_migration" class="anchor-link"></a><a
     <a href="https://issues.apache.org/jira/projects/KAFKA" target="_blank">project JIRA</a> and the "kraft" component.
   </p>
 
-  </p>
-
   <h3>Terminology</h3>
   <ul>
-    <li>Brokers that are in <b>ZK mode</b> store their metadata in Apache ZooKeeper. This is the old mode of handling metadata.</li>
+    <li>Brokers that are in <b>ZK mode</b> store their metadata in Apache ZooKepeer. This is the old mode of handling metadata.</li>
```
Comment (cmccabe) [inline docs/ops.html]: This adds a typo

File: docs/ops.html [hunk 4]
```diff
@@ -3613,23 +3613,22 @@ <h4 class="anchor-heading"><a id="kraft_zk_migration" class="anchor-link"></a><a
...
   In general, the migration process passes through several phases.
+
+  For
   <ul>
-    <li>In the <b>initial phase</b>, all the brokers are in ZK mode, and there is a ZK-based controller.</li>
-    <li>During the <b>initial metadata load</b>, a KRaft quorum loads the metadata from ZooKeeper,</li>
-    <li>In <b>hybrid phase</b>, some brokers are in ZK mode, but there is a KRaft controller.</li>
-    <li>In <b>dual-write phase</b>, all brokers are KRaft, but the KRaft controller is continuing to write to ZK.</li>
-    <li>When the migration has been <b>finalized</b>, we no longer write metadata to ZooKeeper.</li>
+    <li>In the <b>pre-migration phase</b>, all the brokers are in ZK mode, and there is a ZK-based controller. But a KRaft controller has been provisioned and has migration enabled</li>
```
Comment (cmccabe) [inline docs/ops.html]: Change "controller" to "controller quorum"

**Comment by cmccabe** [comment]
Thanks for the PR, @lzyLuke .

Please remove these two lines from "The following features are not fully implemented in KRaft mode":
>     <li>Modifying certain dynamic configurations on the standalone KRaft controller</li>
>    <li>Delegation tokens</li>

**Comment by lzyLuke** [comment]
> Thanks for the PR, @lzyLuke .
> 
> Please remove these two lines from "The following features are not fully implemented in KRaft mode":
> 
> > ```
> > <li>Modifying certain dynamic configurations on the standalone KRaft controller</li>
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> >    * Delegation tokens

I could not find "delegation tokens" related lines. Probably it has been deleted already?

**Review by cmccabe — APPROVED** [review]
LGTM

**Pull request merged**